src/screens/Request/Request.tsx

```diff
@@ -0,0 +1,138 @@
+import Button from '@mui/material/Button';
+import TextField from '@mui/material/TextField';
+import Box from '@mui/material/Box';
+import { Navigate } from 'react-router-dom';
+import { useFormik } from 'formik';
+import * as Yup from 'yup';
+import { IRequest } from '../../models/request.models';
+import { addNewRequest } from '../../service/vacationsRequestService';
+import { Layout } from '../../shared/Layout';
+import { Container } from '@mui/system';
+import Grid from '@mui/material/Grid';
+import { TextareaAutosize } from '@mui/material';
+
+const requestSchema = Yup.object().shape({
+  type: Yup.string().min(2).max(10).required(),
+  startDate: Yup.date().required(),
+  endDate: Yup.date().required(),
+  notes: Yup.string(),
+});
+
+export default function Request() {
```

Comment:
Using in some places function and in other arrow functions to define components it's a bad practice. You should be consistent in the way you define components.
A preferred way to define components it's arrow functions.
please check that point across the app and fix it.